src/shared/use-context-selection/use-manage-inter-param-dependencies.js

```diff
@@ -42,7 +42,9 @@ function useHandleDataSetIdChange() {
         organisationUnits: assignedOrgUnits,
         periodType: dataSetPeriodType,
     } = selectors.getDataSetById(metadata, dataSetId) || {}
-    const setHighlightedFieldId = useSetHighlightedFieldIdContext()
+    const setHighlightedFieldId = useHighlightedFieldStore(
+        (state) => state.setHighlightedField
+    )
```

Comment:
I think we can remove this, if you agree about my comment above (regarding resetting on umount).

Reply:
I don't mind removing it from here, if you insist. But on the other hand, I also think that doing this here actually makes quite a lot of sense too. If we set it to null when the entry field unmounts we'd have to do so conditionally, if `highlighted` is true.

Reply:
This seems no longer necessary as you are setting the highlighted field to null on EntryField unmount, conditional on `highlighted` being true